relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -293,10 +349,9 @@ fn scrub_resource(resource_type: &str, string: &str) -> Option<String> {
             return Some("browser-extension://*".to_owned());
         }
         scheme => {
-            let domain = url
-                .domain()
-                .and_then(|d| normalize_domain(d, url.port()))
-                .unwrap_or("".into());
+            let scrubbed_host = url.host().map(scrub_host).unwrap_or(Cow::Borrowed(""));
+            let domain = concatenate_host_and_port(Some(scrubbed_host.as_ref()), url.port());
```

Comment:
Same as above:
You unwrap the option just to create another option


```suggestion
            let scrubbed_host = url.host().map(scrub_host);
            let domain = concatenate_host_and_port(scrubbed_host.as_deref(), url.port());
```